Tests/UnitTests/Identity/CustomerInfoManagerTests.swift

```diff
@@ -415,3 +378,129 @@ class CustomerInfoManagerTests: TestCase {
     }
 
 }
+
+// iOS 13.0+ only because these tests are async
```

Comment:
Not ideal that we don't run these on iOS 12.x, but a few weeks ago we weren't even running any tests on iOS 12. So I think it's a good compromise, because the tests are _a lot_ simpler.